src/batch/client.ts

```diff
@@ -149,37 +131,28 @@ export class BatchTranscription {
       15 * 60 * 1000, // 15 minutes timeout
     );
 
-    return await this.getJobResult(submitResponse.id, format);
+    return await this.getJobResult(submitResponse.id, format || 'json-v2');
   }
 
-  async createJob({
-    input,
-    fileName,
-    transcription_config,
-    translation_config,
-    output_config,
-    summarization_config,
-    auto_chapters_config,
-    topic_detection_config,
-  }: CreateJobConfig): Promise<CreateJobResponse> {
+  async createJob(
+    input: JobInput,
+    jobConfig: Omit<JobConfig, 'type'>,
```

Comment:
Yeah good shout, we should enforce the non-optionality of `transcription_config` in this case

Reply:
Sorry, to answer your initial question about whether the types: my thought was that we could just enforce the split between different job types at the function level. So `createTranscriptionJob` would accept a config object which requires the fields we need for that type of job (and analogously for alignment), rather than creating a new union type.

I'm generally partial to union types though, so maybe it makes sense. I'm just wary about introducing too many abstractions over the (admittedly quite bare) SDK types